db/rdbms/src/main/java/io/camunda/db/rdbms/write/service/UserTaskWriter.java

```diff
@@ -98,6 +107,22 @@ public void update(final UserTaskDbModel userTaskDbModel) {
               "io.camunda.db.rdbms.sql.UserTaskMapper.insertCandidateGroups",
               userTaskDbModel));
     }
+    executionQueue.executeInQueue(
+        new QueueItem(
+            ContextType.USER_TASK,
+            WriteStatementType.DELETE,
+            userTaskDbModel.userTaskKey(),
+            "io.camunda.db.rdbms.sql.UserTaskMapper.deleteTags",
+            userTaskDbModel.userTaskKey()));
+    if (userTaskDbModel.tags() != null && !userTaskDbModel.tags().isEmpty()) {
+      executionQueue.executeInQueue(
+          new QueueItem(
+              ContextType.USER_TASK,
+              WriteStatementType.INSERT,
+              userTaskDbModel.userTaskKey(),
+              "io.camunda.db.rdbms.sql.UserTaskMapper.insertTags",
+              userTaskDbModel));
+    }
```

Comment:
❓ The tags seem to be immutable, so why is it necessary to delete them and insert them again?